etc/insights-client.spec

```diff
@@ -136,8 +135,11 @@ if ! [ -d "/var/lib/insights" ]; then
 mkdir -m 644 /var/lib/insights
 fi
 
-# symlink the motd file
-ln -snf /etc/insights-client/insights-client.motd /etc/motd.d/insights-client
+#Link motd into place. Only do so if insights-client new install
+if  [ $1 -eq 1  ]; then
+    mkdir -p /etc/motd.d
+    ln -snf /etc/insights-client/insights-client.motd /etc/motd.d/insights-client
+fi
```

Comment:
@gravitypriest This is probably the only change necessary avoid the message reappearing after upgrade.